config/jobs/kubernetes-sigs/jobset/jobset-presubmit-release-0.4.yaml

```
@@ -0,0 +1,174 @@
+presubmits:
+  kubernetes-sigs/jobset:
+  - name: pull-jobset-test-unit-release-0-4
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^release-0.4
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-apps
+      testgrid-tab-name: pull-jobset-test-unit-release-0-4
+      description: "Run jobset unit tests"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        env:
+        - name: GO_TEST_FLAGS
+          value: "-race -count 3"
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        args:
+        - make
+        - test
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
+          requests:
+            cpu: 3
+            memory: 10Gi
+  - name: pull-jobset-test-integration-release-0-4
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^release-0.4
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations: 
+      testgrid-dashboards: sig-apps
+      testgrid-tab-name: pull-jobset-test-integration-release-0-4
+      description: "Run jobset integration tests"
+    spec:
+      containers:
+      - image: public.ecr.aws/docker/library/golang:1.22
+        command:
+        - make
+        args:
+        - test-integration
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
+          requests:
+            cpu: 3
+            memory: 10Gi
+  - name: pull-jobset-test-e2e-main-1-27
```

```suggestion
  - name: pull-jobset-test-e2e-release-0.4-1-27
```

let’s make sure the e2e jobs match the release branch and not main

---

Yeap